.github/workflows/testing.yml

```diff
@@ -0,0 +1,48 @@
+name: Testing
+# in this file we describe the action to execute for the CI in the dev pipeline
+
+# based on default laravel workflow
+# https://github.com/actions/starter-workflows/blob/master/ci/laravel.yml
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
remote the branch master as it is already tested with the workflow for prod

Reply:
Fixed in 333c30c790ddb961ea35498d00a15cf01e0e5b29, `master` only runs from `prod` pipeline now